packages/gestalt/src/Button.js

```diff
@@ -42,7 +42,9 @@ export default function Button(props: Props) {
     white: 'darkGray',
   };
 
-  const classes = classnames(styles.button, {
+  const borderRadius = `borderRadius${inline ? 'Small' : 'Large'}`;
```

Comment:
This feels hacky, can we instead import the Borders.css file and add both the small and large class names to the object down below insde of `classname`. Then for one you use `inline` and for the other use `!inline`

Reply:
I did that as a first pass, not all that crazy about it.  Having one thing set to `inline` and the other set to `!inline` felt a little weird to me.  Also wondering what would happen if PDS added another border-radius requirement, something like borderRadiusMedium

Reply:
If pds adds a third option then both of our methods fail. The string templating just feels very off for creating class names. If you dont like the `inline` `!inline` thing then there is another option of adding a param to `classnames` that looks like `inline ? borders.small : borders.large` since it can take infinite params

Reply:
right, but in one version there is 3 places to update, while in the other the logic is all in one place.  I'm fine adding inline/!inline for now though